TailLight/vfeature.cpp

```diff
@@ -32,9 +32,21 @@ NTSTATUS SetFeatureColor (
     _In_ ULONG     Color
     )
 /*++
+Routine Description:
+
     This routine sets the HID feature by sending HID ioctls to our device.
     These IOCTLs will be handled by HIDUSB and converted into USB requests
     and send to the device.
+
+Arguments:
+
+    Device - Our taillight device.
+
+    Color - A COLORREF value.
+
+Returns:
+
+    Whether everything succeeded or a step failed.
```

Comment:
The argument and return-value documentation above is obvious. There's little point in adding boilerplate documentation like this. You just increase verbosity.